spec/models/post.spec.js

```diff
@@ -15,6 +15,11 @@ describe("Post model", () => {
     expect(post.message).toEqual("some message");
   });
 
+  it.skip("has a user id", () => {
```

Comment:
Why is this skipped?

Reply:
Apparently the last modification we made to this test was not committed. At that point, that test was not working. I have pushed a new commit with that test modified and passing